### PR TITLE
Update Travail d'Equipe

### PR DESCRIPTION
**Pull request by gmoocto**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -110,12 +110,12 @@ <h2>Travail d'équipe</h2>
     <p>Tous les profils sont concernés. C'est un travail d'équipe sous la houlette des <abbr lang="en" title="Product Owners">PO</abbr></p>
```
Comment (elba-octo) [inline index.html:110]:
```suggestion
    <p>Tous les profils sont concernés. C'est un travail d'équipe sous la houlette des <abbr lang="en" title="Product Owners">PO</abbr>.</p>
```

File: index.html [hunk 2]
```diff
@@ -110,12 +110,12 @@ <h2>Travail d'équipe</h2>
...
 
     <h3>
-        La <span lang="en">dream team <abbr>a11y</abbr></span>
+        Les rôles clés
         <small id="competences-clef-a11y-web">Les compétences clef pour l'accessibilité web</small>
     </h3>
     <ul aria-describedby="competences-clef-a11y-web">
         <li>
-            <strong>Designer</strong> formé au design accessible
+            <strong>Designer</strong> formé·e au design inclusif accessible
```
Comment (elba-octo) [inline index.html:118]:
```suggestion
            <strong>Designer</strong> formé·e au design inclusif et accessible
```

File: index.html [hunk 3]
```diff
@@ -124,20 +124,20 @@ <h3>
             <a href="#define-aria"><abbr lang="en" title="Accessible Rich Internet Application">ARIA</abbr></a>
         </li>
         <li>
-            <strong><abbr lang="en" title="Co-Product Owner">CoPO</abbr></strong>
+            <strong lang="en">Product Owner</abbr></strong>
```
Comment (elba-octo) [inline index.html:127]:
```suggestion
            <strong lang="en">Product Owner</strong>
```

File: index.html [hunk 4]
```diff
@@ -110,12 +110,12 @@ <h2>Travail d'équipe</h2>
     <p>Tous les profils sont concernés. C'est un travail d'équipe sous la houlette des <abbr lang="en" title="Product Owners">PO</abbr></p>
 
     <h3>
-        La <span lang="en">dream team <abbr>a11y</abbr></span>
+        Les rôles clés
         <small id="competences-clef-a11y-web">Les compétences clef pour l'accessibilité web</small>
```
Comment (elba-octo) [inline index.html:114]:
```suggestion
        <small id="competences-clef-a11y-web">Les compétences clés pour l'accessibilité web</small>
```

File: index.html (hunk 3 above)
Comment (gmoocto) [reply index.html:127]: 🙈

File: index.html (hunk 2 above)
Comment (gmoocto) [reply index.html:118]: Dans la Refcard on a "design inclusif accessible" 🤔

**Pull request merged**